acceptance/tests/dat_reader.rs

```diff
@@ -12,6 +12,11 @@ fn reader_test(path: &Path) -> datatest_stable::Result<()> {
         path.parent().unwrap().to_str().unwrap()
     );
 
+    // TODO(zach): skip iceberg_compat_v1 test until DAT is fixed
+    if path.ends_with("iceberg_compat_v1/test_case_info.json") {
```

Comment:
We already have a way to skip tests: https://github.com/delta-io/delta-kernel-rs/blob/main/acceptance/src/data.rs#L64

Reply:
whoops lol thanks!